config/application-augustgrades.properties

```diff
@@ -0,0 +1,3 @@
+# Override the get grades date to get from august 1, 2017, 6pm.
```

Comment:
Do we need to create a new file "augustgrades.properties" in GitHub? Is this just for testing purpose?

Reply:
It is just for testing.  I'll try to remove it.